eos-dev-suggests

```diff
@@ -21,7 +21,7 @@ gnome-tweak-tool
 gparted
 jq
 kbd
-linux-tools-3.10
+linux-tools-generic
```

Comment:
One other thought here - since `linux-tools-generic` comes with the `linux` package, this probably isn't needed since I highly doubt we'll accidentally remove `linux` :smile: